core/src/androidTest/java/org/hisp/dhis/android/core/common/CategoryMockFactory.java

```diff
@@ -0,0 +1,23 @@
+package org.hisp.dhis.android.core.common;
+
+
+import android.support.annotation.NonNull;
+
+import org.hisp.dhis.android.core.category.Category;
+
+import java.util.Date;
+
+public class CategoryMockFactory {
```

Comment:
this class is not a factory, it's an [object mother](https://martinfowler.com/bliki/ObjectMother.html), can you refactor it?